cwrapper/arachne_wrapper.cc

```diff
@@ -60,6 +60,23 @@ arachne_wait_termination() {
     Arachne::waitForTermination();
 }
 
+/**
+ * This function is the wrapper for Arachne::createThreadWithClass.
```

Comment:
Probably add documentation about which class is the default thread class and which class is the exclusive one. Or maybe add an enum struct.

Reply:
I've added documentation in the latest commit.